pkg/policy/criteria/matchers.go

```diff
@@ -56,13 +63,21 @@ func matchStringStartsWith(dst *ast.Body, left *ast.Term, right parser.Value) er
 }
 
 func matchStringList(dst *ast.Body, left *ast.Term, right parser.Value) error {
+	str, ok := right.(parser.String)
+	if ok {
+		right = parser.Object{
+			"is": str,
+		}
+	}
+
 	obj, ok := right.(parser.Object)
 	if !ok {
 		return fmt.Errorf("expected object for string list matcher, got: %T", right)
 	}
 
 	lookup := map[string]matcher{
 		"has": matchStringListHas,
+		"is":  matchStringListHas,
```

Comment:
is `is` the same as `has`? I might expect `is` to perform an exact match, but `has` to be more of a "contains" operation.

Reply:
This is matching a list. Do you mean: `is: Smith` should match `["Smith"]` but not `["Smith","Jones"]`.

Reply:
Yes, that is what I would expect.

Reply:
Ok that seems reasonable.